src/style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300px;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 800px;
+  font-style: normal;
+}
```

Comment:
You can use one name for font-family. In separate @font-face you can change only font-weight property and use it in needed selectors

Reply:
Understood. Thank you a lot.